tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilerOptions": {
+    "outDir": "build/dist",
+    "module": "esnext",
+    "target": "es5",
+    "lib": [
+      "es6",
```

Comment:
If you're not targeting NodeJS, you can easily bump up the ECMAScript version in `lib` section to `esnext` and enjoy the goodies of modern JS syntax. There are some caveats — like if you use `Array.prototype.includes` or `Object.fromEntries`, the TS will leave it as is, and you'll need to include polyfill for older browsers.

Reply:
Yes, let's enjoy the goodies of modern JS syntax ;)